src/main/java/gift/service/UserService.java

```diff
@@ -21,6 +21,16 @@ public UserService(H2UserRepository userRepository, JwtProvider jwtProvider) {
         this.jwtProvider = jwtProvider;
     }
 
+    public User getUserById(Long userId) {
+        Optional<User> userFound = userRepository.findById(userId);
+
+        if (userFound.isEmpty()) {
+            throw new InvalidLoginException("해당 이메일이나 비밀번호로 가입된 계정이 없습니다.");
+        }
+
+        return userFound.get();
+    }
```

Comment:
```suggestion
        return userRepository.findById(userId)
                .stream()
                .findFirst()
                .orElseThrow(() -> new InvalidLoginException("해당 이메일이나 비밀번호로 가입된 계정이 없습니다."));
```
- Stream API를 활용해서 위와 같이 표현할 수도 있어요~